pandas/tests/test_join.py

```diff
@@ -192,3 +192,24 @@ def test_inner_join_indexer2():
 
     exp_ridx = np.array([0, 1, 2, 3], dtype=np.int64)
     assert_almost_equal(ridx, exp_ridx)
+
+
+def test_merge_join_categorical_multiindex():
+    # From issue 16627
+    import pandas as pd
```

Comment:
dont' import ``pd``, just add the needed imports at the top.

can you also add an example used an ordered CI